app/src/main/java/sergio/splashbaseviewer/mvp/view/SplashView.java

```diff
@@ -1,32 +1,44 @@
 package sergio.splashbaseviewer.mvp.view;
 
 import android.app.Activity;
-import android.widget.TextView;
+import android.widget.ListView;
+import android.widget.Toast;
 
 import com.squareup.otto.Bus;
 
+import java.util.List;
+
 import butterknife.BindView;
 import butterknife.ButterKnife;
 import sergio.splashbaseviewer.R;
+import sergio.splashbaseviewer.adapters.ImageContainerAdapter;
+import sergio.splashbaseviewer.utils.models.SplashImageResponse;
 
 /**
  * @author s.ruiz
  */
 
 public class SplashView extends ActivityView {
     private final Bus bus;
+    private ImageContainerAdapter adapter;
 
-    @BindView(R.id.tv_splash_main)
-    TextView jsonMain;
+    @BindView(R.id.image_container)
+    ListView imageContainer;
```

Comment:
Its not bad to use ListView, but Android recommended use RecyclerView. Its the improved version of listview. Change this.